### PR TITLE
Static analysis fixes

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: lib/batchVersions.js [hunk 1]
```diff
@@ -70,8 +70,7 @@ export async function create({id, tokenizerId, options}) {
     }
   };
   try {
-    const result = await collection.insertOne(record, database.writeOptions);
-    record = result.ops[0];
```
Comment (mattcollier) [inline lib/batchVersions.js:74]: `record` here was not being used and therefore result was unused.

File: lib/documents.js [hunk 2]
```diff
@@ -156,8 +156,6 @@ export async function register({
       // `externalIdHash`; loop to refresh it
       continue;
     }
-
-    return _getRegistrationRecord({externalIdHash, documentHash});
```
Comment (mattcollier) [inline lib/documents.js:160]: this code is unreachable.

File: lib/tokens.js [hunk 3]
```diff
@@ -361,7 +361,6 @@ async function _createToken({
   const unwrappedSize = _roundToMultipleOf8(
     batchIdSize + 1 + attributes.length);
   const toWrap = new Uint8Array(unwrappedSize);
-  let dv = new DataView(toWrap.buffer, toWrap.byteOffset, toWrap.length);
```
Comment (mattcollier) [inline lib/tokens.js:364]: this `dv` was not used before it was reassigned on L396.

File: lib/tokens.js [hunk 4]
```diff
@@ -446,7 +445,7 @@ async function _parseToken({token}) {
     parsed.payload = base58.decode(payload.substr(1));
 
     if(meta) {
-      if(!(meta && typeof meta === 'string' && meta[0] === 'z')) {
```
Comment (mattcollier) [inline lib/tokens.js:449]: due to L447, `meta` is always truthy here.

File: lib/batchVersions.js (hunk 1 above)
Comment (dlongley) [reply lib/batchVersions.js:74]: I think the problem here is that we weren't returning `record`. So we should change it to do that instead.

File: lib/documents.js (hunk 2 above)
Comment (dlongley) [reply lib/documents.js:160]: Yeah, should be removed.

**Review by dlongley — CHANGES_REQUESTED** [review]
One minor change and then this looks good.

**Review by dlongley — APPROVED** [review]

**Pull request merged**